src/utils/formatAPIObject.ts

```diff
@@ -107,10 +103,7 @@ function matchBlacklist(context: FormatAPIObjectOutputContext | null): boolean {
  * @param key - The key to apprent to the parent array.
  * @returns - A new context.
  */
-function buildContext(
-  context: FormatAPIObjectOutputContext | null,
-  key: any
-): FormatAPIObjectOutputContext | null {
+function buildContext(context: FormatAPIObjectOutputContext | null, key: any): FormatAPIObjectOutputContext | null {
```

Comment:
Mind revert whitespace changes in this file?

Reply:
@songgao I'm not sure what I'm doing wrong but eslint keeps overwriting these files